internal/utils/utils.go

```diff
@@ -337,3 +337,24 @@ func ExtractResourceId(id string) string {
 		return id
 	}
 }
+
+// DefaultTagsHandler converts default_tags attribute to map[string]string
+// supports only string values
+func DefaultTagsHandler(m map[string]interface{}, d *diag.Diagnostics) map[string]string {
```

Comment:
don't think this is used anywhere, can you check?

Reply:
it's unused. The linter should have caught this though. I will remove this in a future PR.